EventFilter/HcalRawToDigi/src/HcalDataFrameFilter.cc

```diff
@@ -26,12 +47,28 @@ namespace HcalDataFrameFilter_impl {
     return es;
   }
 
+  double energySumQIE11(const QIE11DataFrame& df, unsigned int fs, unsigned int ls, const HcalDbService* conditions) {
```

Comment:
same here: better specialize the available template method (the original may need an extra (default = nullptr) argument.